docs/content/app-protect-dos/installation.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Installation with NGINX App Protect Dos
-description: "This document provides an overview of the steps required to use NGINX App Protect Dos with your NGINX Ingress Controller deployment."
+title: Installation with NGINX App Protect DoS
+description: "This document provides an overview of the steps required to use NGINX App Protect DoS with your NGINX Ingress Controller deployment."
 weight: 1800
 doctypes: [""]
 toc: true
 docs: "DOCS-583"
 ---
 
-> **Note**: The NGINX Kubernetes Ingress Controller integration with NGINX App Protect DoS requires the use of NGINX Plus.
+> **Note**: The F5 NGINX Kubernetes Ingress Controller integration with NGINX App Protect DoS requires the use of NGINX Plus.
```

Comment:
doesn't it have to be `F5 NGINX App Protect DoS` and `F5 NGINX Plus` then?

Reply:
The very first mention of the product name on the page requires the "F5" prepend.
This is not required in Notes within the body of the page.  This is not required for every mention of the product names.
The official product names remain "NGINX Ingress Controller" (without 'Kubernetes'), "NGINX Plus", and "NGINX App Protect DoS"

Reply:
@brianehlert I think @lucacome was wondering if this convention applied to the first mention of any product name, as opposed to the product that a given document is written/centered around. I'm not actually sure about this, so I'll check in with my team.

Reply:
Having checked in with my team, it is in fact the first mention of any product name in any document. Pushing an update.